simple/test.js

```diff
@@ -1955,6 +2005,22 @@ exports.test = function (global) {
             name: "aql-sort-all-string",
             params: { func: sortAll, attr: "value6" }
           },
+          {
+            name: "aql-sort-double",
+            params: { func: sortDoubles },
+          },
+          {
+            name: "aql-sort-int1",
+            params: { func: sortIntegers, max: 100  },
+          },
+          {
+            name: "aql-sort-int2",
+            params: { func: sortIntegers, max: 10000  },
+          },
+          {
+            name: "aql-sort-int4",
+            params: { func: sortIntegers, max: 1000000000  },
```

Comment:
We should also add tests for larger ints, because one of the optimizations was specifically for ints with at least 5 bytes.